frontend/src/components/unionCreateComponent.vue

```diff
@@ -135,23 +136,33 @@ export default {
   name: "unionCreate",
   methods: {
     create: function () {
-      UnionApi.postUnion({
-        union: {
-          name: this.name,
-          description: this.description,
-          members_can_invite: this.members_can_invite,
-          icon: "", // If no img don't give it one because it will grap default
-          banner: "",
-        },
-      }).then(() =>
-        router.push({ name: "union-view", params: { unionName: this.name } })
-      );
+      const data = new FormData();
+      data.append('union_id', this.name);
+      this.banner ? data.append("banner", this.banner) : data.append("banner", '');
+      this.icon ? data.append("icon", this.icon) : data.append("icon", '');
+
+      UnionApi.postUnionImages({
```

Comment:
Als je hier nu data mee geeft en niet { data } dan hoef je in postUnionImages ook niet data: data.data te doen.